.github/workflows/ci.yml

```diff
@@ -196,6 +196,13 @@ jobs:
           remote_path: releases/${{ steps.metadata.outputs.version == 'main' && 'latest' || steps.metadata.outputs.version }}
           accelerate: true
           clean: false
+      - name: Generate SHA256 checksums
+        if: ${{ startsWith(github.ref, 'refs/tags/') }}
+        run: |
+          sha256sum .dist/package/${{ matrix.target }}/${{ matrix.task }}/build/cli/* > sha256sums.txt
+          sha256sum hack/mirror/walrus-images.txt >> sha256sums.txt
+          sha256sum hack/mirror/walrus-load-images.sh >> sha256sums.txt
+          sha256sum hack/mirror/walrus-save-images.sh >> sha256sums.txt
```

Comment:
`sed` always includes a useless dirname, e.g. https://github.com/seal-io/terraform-provider-courier/blob/20612f54d1335c3b206ef3e519b7273384f4d8ff/hack/release.sh#L17, we need to trim them.

Reply:
can we find a release action implementation with shasum generation?

Reply:
Updated to use https://github.com/wangzuo/action-release-checksums. There are other implementations like https://github.com/marketplace/actions/generate-checksum, but turns out to include path as well.